### DISCUSSION TITLE
ViTDet

### DISCUSSION THREAD

**Discussion by mohamedettebayo** [roboflow/notebooks#11] [Video ideas]
Would you please make a tutorial on how to use detectron2 projects like ( [ViTDet](https://github.com/facebookresearch/detectron2/tree/main/projects/ViTDet)  because i am doing a new research and i need to learn how to use this ViTDet on google colab for instance segmentation training on my custom roboflow datasets

i would be grateful

**Comment by SkalskiP** [comment]
Hello, @mohamedettebayo 👋🏻! Thanks for creating this discussion. 

So my first question is, does it need to be ViTDet? We quite recently created two Detectrion2 videos:
- Training Mask R-CNN on the custom dataset: https://youtu.be/e8LPflX0nwQ
- Inference OneFormer: https://youtu.be/_Zr1pOi7Chw

If it needs to be ViTDet, then I know that we already selected YouTube video topics for the following two videos. So if nothing unexpected would happen we could do it in three weeks at the earliest.

**Reply by mohamedettebayo** [reply]
I would appreciate it if you make a video specifically about ViTDet because i am having trouble during training with this specific project on google colab when using it on my custom datasets (configuration problem)
So i have no problem with waiting 3 weeks or more as long as you can it 
regards

**Reply by SkalskiP** [reply]
@mohamedettebayo please cast your vote here: https://github.com/roboflow/notebooks/discussions/31
